src/js/pagination/specs/paging-collection-spec.js

```diff
@@ -89,6 +96,28 @@ define(['jquery',
                 }
             ));
 
+            it('doesn\'t send unnecessary parameters in request (total pages, total records)', AjaxHelpers.requests(
+                function (requests) {
+                    // After first fetch the state is updated with returned response
+                    collection.fetch();
+                    server.respond(requests);
+                    collection.fetch();
+                    assertNotInQueryParams(requests, ['total_entries', 'total_pages']);
+
+                    var newCollection = new PagingCollection([], {
+                        state: {
+                            pageSize: 10,
+                            totalPages: 10,
+                            totalRecords: 100
+                        }
+                    });
+
+                    newCollection.url = '/test';
+                    newCollection.fetch();
+                    assertNotInQueryParams(requests, ['total_entries', 'total_pages']);
```

Comment:
I'd prefer that this be a positive test. Can you just use ``AjaxHelpers.expectRequest`` and enumerate exactly the parameters that are expected. This would then verify that these two particular parameters aren't specified, but also that strange future ones don't get added too.